syft/cataloger/cpe_test.go

```diff
@@ -119,19 +159,24 @@ func TestGenerate(t *testing.T) {
 		t.Run(test.name, func(t *testing.T) {
 			actual := generatePackageCPEs(test.p)
 
-			if len(actual) != len(test.expected) {
-				for _, e := range actual {
-					t.Errorf("   unexpected entry: %+v", e.BindToFmtString())
-				}
-				t.Fatalf("unexpected number of entries: %d", len(actual))
+			expectedCpeSet := set.NewStringSet(test.expected...)
```

Comment:
How are you liking this set package so far? Do you see this as being something we could consider using in place of our other internal string set implementations?

Reply:
1000% want to replace ours with this